### PR TITLE
New & Fixed AI VOX Sound Files

### PR DESCRIPTION
**Pull request by Davidj361**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl:
tweak: Made AI VOX sound files smaller sized
fix: Fixed improperly pronounced AI VOX lines
soundadd:  Added a ton of new AI VOX sayings
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

AI does not have enough VOX commands and some of the VOX commands were improperly pronounced.
All the sound files have been compressed with variable bitrate aiming for the lowest quality so the AI VOX sound folder should remain around the same size.
ffmpeg -i FILE -c:a libvorbis -ac 1 -ar 16000 -q:a 0 -speed 0 -threads 8 -y OUT

```
Fixed: go, area, bag, barracks, close, code, cyborgs, down, duct, got, hole, lima, mode, motorpool, 'O', roger, slow, virology, 'W', xenobiology, 'Z'

Added: brig, wing, engineering, hall, hallway, machine, humans, air, atmosphere, dangerous, kitchen, 
bar, lusty, pressure, dormitory, botany, chapel, office, sabotage, atmospherics, warden, detective,
 chief, engineer, multitude, bitcoin, harmful,
many, atmospheric, technician, director, scientist, roboticist, chemist, geneticist, virologist, janitor, bartender, cook, botanist, mime, chaplain, curator, quartermaster, shaft, miner, assistant, lawyer, 
drone, personal, revolution, revolutionary,
cult, cultist, wizard, magic, rune, changeling, lifeform, form, devil, demon, blob, slaughter, murder, clockwork, blood, abduction, kidnapped, kidnapping, entity, death, 
humanoid, lavaland, wire, modification, tampered, change, possible, impossible, riot, dead, section, 
virus, disease, creature, plausible, false, true, freezing, cold, vitals, 
weak, strong, vacuum, unknown, server, rogue, equipment, prison, prisoner, theater, canister, piping, quarantine, internals, space, construct, advised, explosive, bomb, 
formed, created, enabled, disabled, restoration, beware, critical, destruction, questionable, answer, unable, fix, airlock, ugh, ughh

Made file sizes as small as possible.
```

EDIT:
```
Added: oof, or, text, Golem, Moth, Lizard, Plasmaman, Flyman, Zombie, Skeleton, Look, Pathetic, Meat, Bone, Panting, Sweating, Corridors, Challenge, Perfect, Immortal, Doomsday, Factory, Cat, Dog, Honk, When, Put, Table, Free, Megafauna, Nuke, God, apc, eva, sm, hop, cmo, ce, hos, loop, woop, sick, what, why, how, comply, are, he, she, they, them, self, supermatter, crystal, piss, nuclear, stop, cease, pacification, abstain, fauna, silicon, woop, bloop, doop, beep, (censor sfx), dot, wew, wow, epic, awful, long, live, chuckle, jesus, christ, mythic, swhtienoise, shonk, live, live2, sairhorn, require, desire, need, lot, if, very, stay, tech, technology, killed, monkey, animal, corporation, gravity, generator, ian, runtime, corgi, vendor, console, circuit, authentication, disk, beam, unsafe, crate, assistance, bleed, hurt, slip, slippery, clowning, oxygen, nitrogen, carbon, dioxide, library, departures, where

Added tips to announcement help.
```

### CODE REVIEW

**Comment by Davidj361** [comment]
Also, feel free to request some VOX lines to be added.

**Comment by Dorsisdwarf** [comment]
do we have "or" yet?

**Comment by Davidj361** [comment]
No ^

**Comment by PKPenguin321** [comment]
wait, how did you make new ones? i thought we didnt know how the original guy did it

**Comment by Qustinnus** [comment]
theres a repo on github that can make them iirc

**Comment by lordpidey** [comment]
I would suggest adding "Text" to the list, so the AI can say "traitor got green text"

**Comment by ShizCalev** [comment]
@PKPenguin321 I duplicated the original sound effects 1:1 about a year ago after a fair amount of time in audacity playing with various effects, I just needed someone to volunteer to do the VA work for it.

**Comment by lordpidey** [comment]
Wait, so these sounds were originally spoken by a human and then transformed?  I thought they were originally based off of some sort of TTS.

**Comment by 81Denton** [comment]
Great job! If it's possible I have some requests: The name of ingame races, some words of SHODAN's speech and other misc words.

>**Races**: Golem-s, Moth-s, Lizard-s, Plasmaman/Plasmamen, Flyman/Flymen, Zombie-s, Skeleton-s
**SHODAN's speech**: Look, Pathetic, Meat, Bone, Panting, Sweating, Corridors, Challenge, Perfect, Immortal 
**Malf**: Doomsday, Factory
**Misc**: Cat, Dog, Honk, When, Put, Table, Free, Megafauna, Nuke, God

**Comment by Dieknon** [comment]
chucklefuck

**Comment by nervere** [comment]
Here's a list I created. You got most of the important ones I had written down but here's some more for what it's worth. I think I eliminated most of the repeats.

``` very, dangerous, if, stay, atmospherics, atmosphere, tech, technology, require, desire, also, need, lot, killed, dead, monkey, animal, corporation, gravity, generator, ian, runtime, cat, dog, corgi, vendor, console, circuit, nuclear, authentication, disk, beam, canister, unsafe, crate, assistance, blood, bleed, hurt, slip, slippery, clowning, honk, pressure, oxygen, nitrogen, carbon, dioxide, library, departures ```

**Comment by Davidj361** [comment]
Alright, I think it's done now. I processed all requests so far. Though I added some sound effects: sbeep, shonk, sairhorn, and swhitenoise. I can remove them if it bothers the maintainers.

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 1]
```diff
@@ -150,53 +188,79 @@ GLOBAL_LIST_INIT(vox_sounds, list("abortions" = 'sound/vox_fem/abortions.ogg',
 "condom" = 'sound/vox_fem/condom.ogg',
 "confirmed" = 'sound/vox_fem/confirmed.ogg',
 "connor" = 'sound/vox_fem/connor.ogg',
+"console2" = 'sound/vox_fem/console2.ogg',
+"console" = 'sound/vox_fem/console.ogg',
```
Comment (lordpidey) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Which one is which?  I imagine one is console as in device, and the other is console as in to provide emotional support.

Comment (Davidj361) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: console = the machine
console2 = to care for someone

**Comment by 81Denton** [comment]
That's a lot of work. Thank you!

**Comment by Cyberboss** [comment]
could you give us a before and after of the vox folder size please

**Comment by Davidj361** [comment]
>could you give us a before and after of the vox folder size please
Before:
Size: 5.01 MB
Size on Disk: 6.14 MB

After:
Size: 5.88 MB
Size on Disk: 7.71 MB

**Review by Cyberboss — APPROVED** [review]

**Comment by DaxDupont** [comment]
Hey can you hit me up on discord Dax Dupont#6320 or soyuz on IRC, I have some questions thanks!

**Comment by BeeSting12** [comment]
@Davidj361 
I need a list of all the sound effects, ie airhorn, honk sounds, beeping, etc. They're very annoying and I'd like to remove them.

**Comment by Davidj361** [comment]
@BeeSting12 
>I need a list of all the sound effects, ie airhorn, honk sounds, beeping, etc. They're very annoying and I'd like to remove them.

sairhorn, shonk, swhitenoise (this one was requested by antur)

**Comment by PKPenguin321** [comment]
>I think the AI should have some sound effect as an alert.

WOOP
WOOP

**Comment by Davidj361** [comment]
>WOOP
WOOP

I honestly tried my best replicating the Half-life VOX "woop", but I couldn't do it.

**Comment by N3X15** [comment]
In the future, please try to PR any beneficial changes like these upstream to https://github.com/N3X15/ss13-vox so everyone else can benefit.  Thanks.

**Pull request merged**